package.json

```diff
@@ -8,8 +8,17 @@
     "node": ">=6.13.0"
   },
   "scripts": {
+    "server": "nodemon server/server.js"
   },
   "dependencies": {
+    "body-parser": "^1.19.0",
+    "cors": "^2.8.5",
+    "express": "^4.17.1",
+    "http-proxy": "^1.18.1",
```

Comment:
if you end up doing the middleware for proxy, let us know how well it works!

Reply:
^^ ditto